kibana-reports/server/model/index.ts

```diff
@@ -123,18 +136,7 @@ export const reportDefinitionSchema = schema.object({
     ),
   }),
 
-  delivery: schema.object({
-    delivery_type: schema.oneOf([
-      schema.literal(DELIVERY_TYPE.kibanaUser),
-      schema.literal(DELIVERY_TYPE.channel),
-    ]),
-    delivery_params: schema.conditional(
-      schema.siblingRef('delivery_type'),
-      DELIVERY_TYPE.kibanaUser,
-      kibanaUserSchema,
-      channelSchema
-    ),
-  }),
+  delivery: deliverySchema,
```

Comment:
Can the delivery type be inferred from the number of items in the recipients so that in the future (when multiple channels are supported), index need not be migrated to new schema?

Reply:
That's a good point. Will address in future PR.